Documentation/General/productPlanning/requirements.tex

```diff
@@ -0,0 +1,146 @@
+
+\subsection{Functional requirements}
+For the product MoodCat, the requirements regarding functionality and service are grouped under the \emph{Functional requirements}.
+Within these functional requirements, four categories can be identified using the \emph{MoSCoW} model \footnote{\url{http://en.wikipedia.org/wiki/MoSCoW_method}} for prioritizing requirements:
+
+\subsubsection{Must haves}
+\begin{itemize}
+\item The user shall interact with the product via a web interface.
+
+\item The web interface shall show a screen to log in to the application.
+
+\item The login mechanism shall be managed by SoundCloud OAuth keys ("Log in with SoundCloud").
+
+\item The login mechanism shall show an error page when the user does not authorize MoodCat to have access to their SoundCloud data.
+
+\item The web interface shall show a first-use tutorial upon logging in succesfully for the first time.
+
+\item The web interface shall show the user a page where they can select their current mood.
+
+\item The web interface shall show the following moods:
+		\begin{itemize}
+		\item Nervous
+		\item Exiting
+		\item Happy
+		\item Pleasing
+		\item Relaxing
+		\item Peaceful
+		\item Calm
+		\item Sleepy
+		\item Bored
+		\item Sad
+		\end{itemize}
+	  These moods are defined by the Arousal/Valence model\footnote{Y. Chang and H.H. Chen. Music Emotion Recognition. Boca raton, State:
+CRC Press, 2011.}.
+
+\item The system shall serve the user a list of rooms, ordered by how close they match the given mood.
+
+\item The web interface shall present the user with a room when it is chosen.
+	
+\item The system shall stream the music from the SoundCloud platform using the SoundCloud API\footnote{\url{https://developers.soundcloud.com/docs/api/guide}}
```

Comment:
The requirement is that the platform is able to stream music to the client. The implementation does not matter, and for a fact, is potential subject to change.